src/pages/create.tsx

```diff
@@ -0,0 +1,161 @@
+import { useState } from "react";
+import { useAuth } from "../../utils/firebase/auth";
+import { Text, Button, Input } from "../components/atomic";
+import { createProgram } from "../generated/graphql";
+import TitledInput from "../components/TitledInput";
+
+const BlobCircle = () => {
+  const sizes = "h-24 w-24 md:h-64 md:w-64 lg:h-80 lg:w-80";
+  return (
+    <div
+      className={`${sizes} rounded-full bg-skyblue overflow-hidden flex justify-center items-end pointer-events-none`}
+    >
+      <img src="/static/HappyBlobs.svg" className="w-11/12 select-none" />
+    </div>
+  );
+};
+
+const SignUpPage = () => {
+  const [stage, setStage] = useState(0);
+  const [programName, setProgramName] = useState("");
+  const [programLogo, setProgramLogo] = useState("");
+  const [programIdentifier, setProgramIdentifier] = useState("");
+  const [displayError, setError] = useState("");
+
+  const stepOne = () => {
+    return (
+      <div className="flex flex-col w-full px-4 py-12 md:w-120">
+        <Text h1 b>
+          Create your mentorship!
+        </Text>
+        <div className="h-6" />
+
+        <div className="flex w-full">
+          <TitledInput
```

Comment:
I would just do input and text separately to make it look more like the figma here